app/routes.py

```diff
@@ -1,2 +1,191 @@
-from flask import Blueprint
+import re
+from flask.helpers import make_response
 
+from werkzeug.datastructures import HeaderSet
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, request, jsonify
+import requests, os, json
+from datetime import datetime
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+BOT_TOKEN = os.environ.get('BOT_TOKEN')
+
+@task_bp.route("", methods=["POST", "GET"])
+def handle_tasks():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return jsonify({"details": "Invalid data"}), 400
+        new_task = Task(
+            title= request_body["title"],
+            description= request_body["description"],
+            completed_at = request_body["completed_at"]
+        )
+        db.session.add(new_task)
+        db.session.commit()
+        if new_task.completed_at:
+            new_task.is_complete = True
+        
+        response = {}
+        response["task"] = new_task.to_dict()
+        return jsonify(response), 201
+
+    elif request.method == "GET":
+        sort_query = request.args.get("sort")
+        if sort_query:
+            if sort_query == "asc":
+                tasks = Task.query.order_by(Task.title).all()
+            elif sort_query == "desc":
+                tasks = Task.query.order_by(Task.title.desc()).all()
+        else: 
+            tasks = Task.query.all()
+        
+        response_body = []
+        for task in tasks:
+            response_body.append(task.to_dict())
```

Comment:
List comprehension would be handy in this case:

```suggestion
        response_body = [ task.to_dict() for task in tasks]
```